src/mito2/src/worker/handle_open.rs

```diff
@@ -62,6 +62,7 @@ impl<S: LogStore> RegionWorkerLoop<S> {
             self.scheduler.clone(),
         )
         .region_dir(&request.region_dir)
+        .options(request.options)
```

Comment:
Will there be a situation where the user repeatedly opens the region with different options?

Reply:
Currently, reopening a region with different options is allowed.